amivapi/studydocs/endpoints.py

```diff
@@ -80,6 +81,11 @@
                 'maxlength': 30,
                 'nullable': True,
                 'type': 'string'
+            },
+            'coursesemester': {
+                    'maxlength': 5,
+                    'nullable': True,
+                    'type': 'string'
```

Comment:
Well the idea is that students can upload their documents at any time and link it to the semester where they had taken the course and had made this document. This can happen during the semester as much as it can happen when a Master student digging through his old documents finds a document which might be helpful and decides to upload it.